lib/reline.rb

```diff
@@ -457,8 +460,8 @@ def ambiguous_width
   def_single_delegator :line_editor, :byte_pointer, :point
   def_single_delegator :line_editor, :byte_pointer=, :point=
 
-  def self.insert_text(*args, &block)
-    line_editor.insert_text(*args, &block)
+  def self.insert_text(text)
+    line_editor.insert_pasted_text(text)
```

Comment:
Should we address the difference in naming in the future? 

Also, could stop taking a block be a breaking change to some users? (I didn't find such usage with either `Reline.insert_text` or `Readline.insert_text` during a quick search on GH tho)

Reply:
It won't be a breaking change. `line_editor.insert_text(text)` only accepts one String argument.
Passing a block to a method that does not use a block does not raise any error.